packages/app-builder-lib/src/index.ts

```diff
@@ -89,6 +89,9 @@ export function build(options: PackagerOptions & PublishOptions, packager: Packa
       }
 
       for (const newArtifact of newArtifacts) {
+        if (buildResult.artifactPaths.includes(newArtifact)) {
+          continue;
```

Comment:
Let's add a log here for the console:
```
log.warn({ newArtifact }, "skipping publish of artifact, already published")
```

Reply:
Good call, updated and merged new changes, please review.